test/integration/integration.test.js

```diff
@@ -0,0 +1,216 @@
+    var loadedConfig = wrapper.loadConfig('./test/integration/config.real.test.json')
+    if (!loadedConfig) {
+      console.log('No config: test/config.real.test.json was not found, no test credentials to use')
+    }
+    wrapper.credsSecret = 'BANANANAANANANANAN'
+    wrapper.auth((error, results) => {
+      if (error) console.log(error)
+      results.should.equal(false)
+      wrapper.authorizedTimestamp.should.equal(false)
+      done()
+    })
+  })
+
+  it('Should request a single full bib record', (done) => {
+    var loadedConfig = wrapper.loadConfig('./test/integration/config.real.test.json')
+    if (!loadedConfig) {
+      console.log('No config: test/config.real.test.json was not found, no test credentials to use')
+    }
+    wrapper.auth((errorAuth, results) => {
+      if (errorAuth) console.log(errorAuth)
+      wrapper.requestSingleBib('17292415', (errorBibReq, results) => {
+        if (errorBibReq) console.log(errorBibReq)
+        results.data.total.should.equal(1)
+        done()
+      })
+    })
+  })
+
+  it('Should request the items associated with a bib id - Two items', (done) => {
+    var loadedConfig = wrapper.loadConfig('./test/integration/config.real.test.json')
+    if (!loadedConfig) {
+      console.log('No config: test/config.real.test.json was not found, no test credentials to use')
+    }
+    wrapper.auth((errorAuth, results) => {
+      if (errorAuth) console.log(errorAuth)
+      wrapper.requestBibItems('17292415', (errorBibReq, results) => {
+        if (errorBibReq) console.log(errorBibReq)
+        results.data.total.should.equal(2)
+        done()
+      })
+    })
+  })
+
+  it('Should request the items associated with a bib id - A lot of items', (done) => {
+    var loadedConfig = wrapper.loadConfig('./test/integration/config.real.test.json')
+    if (!loadedConfig) {
+      console.log('No config: test/config.real.test.json was not found, no test credentials to use')
+    }
+    wrapper.auth((errorAuth, results) => {
+      if (errorAuth) console.log(errorAuth)
+      wrapper.requestBibItems('10568020', (errorBibReq, results) => {
+        if (errorBibReq) console.log(errorBibReq)
+        results.data.total.should.equal(1029)
+        done()
+      })
+    })
+  })
+
+  it('Should request the items associated with a bib id - A lot of items 2', (done) => {
+    var loadedConfig = wrapper.loadConfig('./test/integration/config.real.test.json')
+    if (!loadedConfig) {
+      console.log('No config: test/config.real.test.json was not found, no test credentials to use')
+    }
+    wrapper.auth((errorAuth, results) => {
+      if (errorAuth) console.log(errorAuth)
+      wrapper.requestBibItems('14628261', (errorBibReq, results) => {
+        if (errorBibReq) console.log(errorBibReq)
+        results.data.total.should.equal(813)
+        // console.log(JSON.stringify(results, null, 2))
+        console.log(JSON.stringify(results.data))
```

Comment:
Remove logs unless you need them.

Reply:
I know a lot of it is copying over existing code, but feel free to update it.

Reply:
I've removed the superfluous console logs and commented code